src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlFullTextSearchMethodTranslator.cs

```diff
@@ -41,14 +41,37 @@ public Expression Translate(MethodCallExpression e)
 
             if (e.Method.DeclaringType == typeof(NpgsqlFullTextSearchDbFunctionsExtensions)
                 && SqlNameByMethodName.TryGetValue(e.Method.Name, out var sqlFunctionName))
-                return new SqlFunctionExpression(sqlFunctionName, e.Method.ReturnType, e.Arguments.Skip(1));
+                return TryTranslateDbFunctionExtension(e, sqlFunctionName);
 
             if (e.Method.DeclaringType == typeof(NpgsqlFullTextSearchLinqExtensions))
                 return TryTranslateOperator(e) ?? TryTranslateFunction(e);
 
             return null;
         }
 
+        [CanBeNull]
+        static Expression TryTranslateDbFunctionExtension([NotNull] MethodCallExpression e, [NotNull] string sqlFunctionName)
+        {
+            switch (e.Method.Name)
+            {
+                case nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsVector) when e.Arguments.Count == 3:
+                case nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PlainToTsQuery) when e.Arguments.Count == 3:
+                case nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PhraseToTsQuery) when e.Arguments.Count == 3:
+                case nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsQuery) when e.Arguments.Count == 3:
+                    return new SqlFunctionExpression(
+                        sqlFunctionName,
+                        e.Method.ReturnType,
+                        new[]
+                            {
+                                new ExplicitStoreTypeCastExpression(e.Arguments[1], typeof(string), "regconfig")
+                            }
+                            .Concat(e.Arguments.Skip(2)));
```

Comment:
Since these functions are pre-checked to have three args, it may be cleaner to just access `e.Arguments[2]` in the array constructor above.

Reply:
Done. Force-pushed. @austindrenski